Dockerfile

```diff
@@ -126,5 +126,19 @@ RUN set -e \
 		exit 1; \
 	fi
 
+# Security improvements:
+# Remove server banner
+RUN sed -i "s/\    <Connector\ port=\"8080\"\ protocol=\"HTTP\/1.1\"/\    <Connector\ port=\"8080\"\ protocol=\"HTTP\/1.1\"\n\               Server=\" \"/g" /usr/local/tomcat/conf/server.xml && \
```

Comment:
Please add this to the end of the `RUN` line that contains `yum clean all` (obvs. add the `&&` ;-) ) to cut down on the number of layers.

Reply:
I have moved the whole block of commands into the above RUN command that already ends with ```yum clean all``` used for installing tomcat :), thank you for your time and suggestion.